yarn-project/end-to-end/src/e2e_epochs/epochs_empty_blocks.test.ts

```diff
@@ -61,12 +61,15 @@ describe('e2e_epochs/epochs_empty_blocks', () => {
         `Reached PENDING L2 block ${epochTargetBlockNumber}, proving should now start, waiting for PROVEN block to reach ${provenBlockNumber}`,
       );
       await test.waitUntilProvenL2BlockNumber(provenBlockNumber, 120);
-      expect(Number(await rollup.getProvenBlockNumber())).toBe(provenBlockNumber);
+      expect(Number(await rollup.getProvenBlockNumber())).toBeGreaterThanOrEqual(provenBlockNumber);
       logger.info(`Reached PROVEN block number ${provenBlockNumber}, epoch ${epochNumber} is now proven`);
       epochNumber++;
 
       // Verify the state syncs
       await test.waitForNodeToSync(provenBlockNumber, 'finalised');
+      // TODO(MW): While waiting for the node to sync, the proven block number can overtake what we expect,
+      // hence updated below - TODO should rework timings to ensure this test covers all blocks?
+      provenBlockNumber = (await test.context.aztecNode.getWorldStateSyncStatus()).finalisedBlockNumber;
```

Comment:
I found this was failing because a) `rollup.getProvenBlockNumber()` was much further ahead than expected and b) because while waiting for `await test.waitForNodeToSync(provenBlockNumber, 'finalised');`, the proven blocks moved so far ahead that the `provenBlockNumber` was part of the old removed historic blocks below ( <= `expectedBlockRemoved`).
To mitigate this I temporarily updated `provenBlockNumber` but would probably need to rework this test or the framework a bit better to account for faster blocks?